utils/getExchangeLinks.ts

```diff
@@ -0,0 +1,60 @@
+interface TokenExchangeMap {
+  [tokenSymbol: string]: number;
+}
+
+interface ExchangeNameMap {
+  [exchangeType: number]: string;
+}
+
+export enum EXCHANGES {
+  UNISWAP,
+  BALANCER,
+}
+
+export const EXCHANGE_NAMES: ExchangeNameMap = {
+  [EXCHANGES.UNISWAP]: "Uniswap",
+  [EXCHANGES.BALANCER]: "Balancer",
+};
+
+// Returns a function for a given exchange that can be used to retrieve an exchange's specific swap URL for a chosen token.
+interface ExchangeLinkMap {
+  [exchangeEnum: number]: (params: any) => string;
+}
+
+// A mapping of synthetic tokens to the exchange where they can be traded.
```

Comment:
```suggestion
// A mapping of synthetic tokens to the exchange where they and their collateral can be traded.
```

Reply:
I committed this locally so ignoring the GUI commit.